### DISCUSSION TITLE
Les fiches ont été fusionnées avec le code du générateur de site

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#8] [Announcements]
Comme annoncé, une version des fiches au format web générées avec [GatsbyJS](https://www.gatsbyjs.com/) existe. Vous pouvez en voir le rendu sur ce site (temporaire et actualisé manuellement) https://cnumr-wordpress-best-practices.netlify.app/.

Afin d'optimiser la génération de ces fiches, GitHub, grâce à l'intégration continue des [GitHub Actions](https://fr.github.com/features/actions) permet de les publier sur [GitHub Pages](https://pages.github.com). Malheureusement, GitHub ne permet l'ouverture des Pages que sur des repo payant ou des repo public. Ce qui m'amène à la question, **peut-on ouvrir ce repo pour avoir une publication automatiser ou doit-on le laisser privé pour ne pas se faire "piquer" nos fiches ?**

Merci de donner votre avis

@hrenaud

**Comment by florinesueur** [comment]
Personnelement, j'imaginais un repo public. Peut-être une licence CC BY-NC-SA 4.0 du coup. Vous en pensez quoi ?

**Reply by dNicolle** [reply]
@florinesueur On l'ouvrira bien sur, lorsqu'on l'aura finalisé.
Ensuite pour les licence, je n'y connait absolument rien :/

Ensuite, la mise en place d'un site, c'est surtout parce qu'il faut le rendre accessible aux non dev. et Github n'est pas l'endroit le plus pertinent pour eux.
